src/components/Organisms/pages/page_home/SubjectTeacher.jsx

```diff
@@ -0,0 +1,84 @@
+import { TbMathSymbols, TbMessageLanguage } from "react-icons/tb";
+import { SlChemistry } from "react-icons/sl";
+import { Icon } from "../../../molecules/Icon";

```

Comment:
I was thinking that we should standarize the way we do imports, because right now we are combining relative imports (the one that locates the files corresponding to the current directory) and absolute imports (the ones that comes from the root). What do you think? I think we should go for absolute imports on every import.

The colors_subjects and the subjects I think are constants right? we could use upper_case characters for those, like "COLOR_SUBJECTS" and "SUBJECTS"

Also I've seen that in javascript is more common to use camelcase for variables that are not constant like in the lines 49, 52 and 53